sopel_waifu/waifu.json

```
@@ -2444,6 +2444,8 @@
         "Rani VIII",
         "Ryougi Mana",
         "Ryougi Shiki (F/GO)",
+        "Ryougi Shiki (Assassin)",
+        "Ryougi Shiki (Saber)",
```

```suggestion
        "Ryougi Shiki (Assassin) (F/GO)",
        "Ryougi Shiki (Saber) (F/GO)",
```

Without this, these new items won't appear in the fgowaifu command you like so much.